cdisc_rules_engine/models/dictionaries/whodrug/whodrug_terms_factory.py

```diff
@@ -28,7 +29,9 @@ def __init__(self, data_service: DataServiceInterface):
             WhodrugFileNames.INA_FILE_NAME.value: AtcText,
         }
 
-    def install_terms(self, directory_path: str) -> Dict[str, List[BaseWhoDrugTerm]]:
+    def install_terms(
+        self, directory_path: str
+    ) -> Dict[str, Dict[str, BaseWhoDrugTerm]]:
```

Comment:
We need to have the same return type in both of the factories: MedDra and WhoDrug. This will allow us depend on the interface, not on the implementation. Having WhoDrug factory interface changed, we also need to refactor MedDra factory. Otherwise, my suggestion would be to rollback WhoDrug factory.

Reply:
Meddra already returns a dictionary of terms